### DISCUSSION TITLE
Empty OON Allowed Amount File

### DISCUSSION THREAD

**Discussion by jamessiefke** [CMSgov/price-transparency-guide#234] [Q&A] [answered]
Based on the latest update, is the expectation that if no data does not exist for a given HIOS segment that we are to produce an empty OON file for that month?

**Comment by shaselton-usds** [comment, marked as answer]
That's correct. 

Just in case you missed it (or anyone else), the file would look roughly like this: https://github.com/CMSgov/price-transparency-guide/blob/2df1618b6fe29f12cdcf55e81e5707da29c6608c/examples/allowed-amounts/multiplan-empty-example.json

**Reply by jamessiefke** [reply]
shaselton,
Based on the new structure, it appears that you can report data based on HIOS-5 but list out all impacted HIOS-14.  Is that correct?  If so, what happens if there is a plan with no data?  Would we need to break that out into an empty file or just have that plan listed in the reporting_plans section of the JSON?

**Comment by tdfow** [comment]
#79
